spec/datadog/appsec/response_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'securerandom'
```

Comment:
Minor: I think it's better to fake the value instead. Input parameters should be known to us in order to verify the output result.